builder/comp-builder.nix

```diff
@@ -162,6 +162,12 @@ let
   exeName = componentId.cname + exeExt;
   testExecutable = "dist/build/${componentId.cname}/${exeName}";
 
+  # Pass -j1 if cross-compiling to work around
+  # https://gitlab.haskell.org/ghc/ghc/issues/18024
+  jobCount = if stdenv.hostPlatform != stdenv.targetPlatform
```

Comment:
But this only happens for the llvm backend, right?

The window cross compiler we use uses the native code gen backend. So it would not be affected by my understanding?

Reply:
The compiler might not fail, but you may get incorrect code, since the `Unique`s used throughout GHC will not be fresh when using a stage 1 compiler linked `-threaded` in parallel.

I happened to notice with the LLVM backend since we were passing invalid IR to `opt` due to the duplicate `Unique`s.

I assume the windows cross compiler is stage 1 too?